### DISCUSSION TITLE
ElectroCardiogram support

### DISCUSSION THREAD

**Discussion by baptooo** [agencyenterprise/react-native-health#45] [Ideas]
Hello ! thank you for the wonderful work 👏 

I was wondering if you were planning to support ElectroCardiogram APIs from HealthKit in order to retrieve ECG signal ?

Thank you !

**Comment by macelai** [comment]
Hey @baptooo, we don't have a roadmap for this. But feel free to contribute with the project!

**Comment by vvazzim** [comment]
Hey @baptooo im on the same project can i ask you few questions about ElectroCardiogram APIs?
